pisa/stages/xsec/pi_genie_sys.py

```diff
@@ -0,0 +1,230 @@
+            'GENIE_quad_fit_CV1uBY',
+            'GENIE_linear_fit_CV2uBY',
+            'GENIE_quad_fit_CV2uBY',            
+            'GENIE_x',            
+            'GENIE_y',            
+            'GENIE_A',            
+        )
+        # what keys are added or altered for the outputs during apply
+        output_apply_keys = (
+            'weights',
+        )
+
+        # init base class
+        super(pi_genie_sys, self).__init__(
+            data=data,
+            params=params,
+            expected_params=expected_params,
+            input_names=input_names,
+            output_names=output_names,
+            debug_mode=debug_mode,
+            input_specs=input_specs,
+            calc_specs=calc_specs,
+            output_specs=output_specs,
+            input_apply_keys=input_apply_keys,
+            output_apply_keys=output_apply_keys,
+        )
+
+        assert self.input_mode is not None
+        assert self.calc_mode is None
+        assert self.output_mode is not None
+
+    @profile
+    def apply_function(self):
+        genie_ma_qe = self.params.Genie_Ma_QE.m_as('dimensionless')
+        genie_ma_res = self.params.Genie_Ma_RES.m_as('dimensionless')
+        genie_ahtby = self.params.Genie_AhtBY.m_as('dimensionless')
+        genie_bhyby = self.params.Genie_BhtBY.m_as('dimensionless')
+        genie_cv1uby = self.params.Genie_CV1uBY.m_as('dimensionless')
+        genie_cv2uby = self.params.Genie_CV2uBY.m_as('dimensionless')
+        nu_diff_dis = self.params.nu_diff_DIS.m_as('dimensionless')
+        nubar_diff_dis = self.params.nubar_diff_DIS.m_as('dimensionless')
+        hadron_dis = self.params.hadron_DIS.m_as('dimensionless')
+        a_scale_dis = self.params.A_scale_DIS.m_as('dimensionless')
+
+        for container in self.data:
+            apply_genie_sys(
+                genie_ma_qe,
+                container['GENIE_linear_fit_MaCCQE'].get(WHERE),
+                container['GENIE_quad_fit_MaCCQE'].get(WHERE),
+                genie_ma_res,
+                container['GENIE_linear_fit_MaCCRES'].get(WHERE),
+                container['GENIE_quad_fit_MaCCRES'].get(WHERE),
+                genie_ahtby,
+                container['GENIE_linear_fit_AhtBY'].get(WHERE),
+                container['GENIE_quad_fit_AhtBY'].get(WHERE),
+                genie_bhyby,
+                container['GENIE_linear_fit_BhtBY'].get(WHERE),
+                container['GENIE_quad_fit_BhtBY'].get(WHERE),
+                genie_cv1uby,
+                container['GENIE_linear_fit_CV1uBY'].get(WHERE),
+                container['GENIE_quad_fit_CV1uBY'].get(WHERE),
+                genie_cv2uby,
+                container['GENIE_linear_fit_CV2uBY'].get(WHERE),
+                container['GENIE_quad_fit_CV2uBY'].get(WHERE),
+
+                out=container['weights'].get(WHERE),
+            )
+
+            # Differential xsec systematic
+            diff_dis = nu_diff_dis
+            diff_norm = 1.- (1.6525*diff_dis)
```

Comment:
There should be no non-obvious "magic" numbers in code without comment. Please provide some explanation what this is, where it comes from (a specific reference), just something so I can check if `1.6525` is correct and understand where it comes from.

Maybe it would suffice to put a reference at the top of the file in the module-level docstring, but just make sure it's something that makes it at least somewhat easy for others to verify all the numbers in this code.